admin/class-yoast-notification-center.php

```diff
@@ -171,7 +214,10 @@ public function clear_dismissal( $notification ) {
 		}
 
 		// Remove notification dismissal for all users.
-		$deleted = delete_metadata( 'user', 0, $dismissal_key, '', true );
+		$deleted = delete_metadata( 'user', 0, $wpdb->get_blog_prefix() . $dismissal_key, '', true );
+
+		// Delete unprefixed user meta too for backward-compatibility.
+		$deleted = delete_metadata( 'user', 0, $dismissal_key, '', true ) || $deleted;
```

Comment:
Not sure if this should be taken into account as a 'success' factor, as this is cleaning up old data.

Reply:
I think it should as the old data would still have effect. Also because it's using OR, it is essentially the same behavior as before and not too strict - it would be if both calls were required to return true.